.gitignore

```diff
@@ -1,11 +1,12 @@
 *.m~
 mym/
-*.mltbx
 *.env
 notebook
 *getSchema.m
-docker-compose.yml
+docker-compose.yaml
```

Comment:
I'd say also add `docker-compose.yml`?

Reply:
Thanks @eywalker. Silly oversight. Should be updated now.